src/main/java/pvs/app/service/GithubApiService.java

```diff
@@ -74,6 +76,26 @@ private void setGraphQlGetIssuesTotalCountQuery(String owner, String name) {
         this.graphQlQuery = graphQl;
     }
 
+    private void setGraphQlGetPullRequestQuery(String owner, String name) {
+        Map<String, Object> graphQl = new HashMap<>();
+        graphQl.put("query", "{repository(owner: \"" + owner + "\", name:\"" + name + "\") {" +
+                "pullRequests (first: 100) {" +
```

Comment:
Suggest extracting this number(`100`) to an isolated variable.

Reply:
Include all the other functions in `GithubApiService`?